src/journal/journald-server.c

```diff
@@ -2177,6 +2177,8 @@ void server_done(Server *s) {
         free(s->tty_path);
         free(s->cgroup_root);
         free(s->hostname_field);
+        free(s->runtime_storage.path);
+        free(s->system_storage.path);
```

Comment:
The last commit fixes https://github.com/systemd/systemd/issues/5516. I think the commit message should be updated.